### PR TITLE
[Admin] Add a link on the IP modal to search for a given IP address accross all users.

### PR DESCRIPTION
**Pull request by Cryptanalyse**
Add the possibility to click on a user IP on the admin to search for this IP, as if if were entered in the search field.

### CODE REVIEW

File: CTFd/themes/admin/templates/modals/users/addresses.html [hunk 1]
```diff
@@ -12,7 +12,11 @@
 			<tbody>
 			{% for addr in addrs %}
 				<tr>
-					<td class="text-center">{{ addr.ip }}</td>
+					<td class="text-center">
+						<a href="{{ url_for('admin.view') }}/users?field=ip&q={{ addr.ip }}">
```
Comment (ColdHeat) [inline CTFd/themes/admin/templates/modals/users/addresses.html:16]: You should be able to do this entirely with a url_for call. Something like `url_for('admin.users', field='ip', q=addr.ip)`.

**Review by ColdHeat — CHANGES_REQUESTED** [review]
If you can make this change I will merge

**Comment by Cryptanalyse** [comment]
Done! Indeed, this is cleaner, thanks!

**Comment by ColdHeat** [comment]
Made a few other changes but thanks for adding this I think it is a good feature.

**Pull request merged**